### PR TITLE
Fix URLs for links to npm packages

### PR DESCRIPTION
**Pull request by firefoxic**
> Which issue, if any, is this issue related to?

I hope I've understood the [request](https://github.com/stylelint/awesome-stylelint/pull/62#issuecomment-1865303274) correctly.

> Is there anything in the PR that needs further explanation?

No, it's self-explanatory.

### CODE REVIEW

**Review by ybiquitous — COMMENTED** [review]
Thanks for opening the pull request! If no other downsides, can you update other links to refer npmjs.com, too? I think it should be more consistent.

**Comment by firefoxic** [comment]
I wanted to make it prettier. Didn't happen... 🤭

**Comment by firefoxic** [comment]
I clicked through all the links, and found some deprecated packages among them. I changed them to new ones.

File: README.md [hunk 1]
```diff
@@ -67,69 +67,69 @@
...
-- [stylelint-declaration-strict-value](https://github.com/AndyOGo/stylelint-declaration-strict-value) - Enforce variables, functions or keywords for the value of specific properties.
-- [stylelint-file-max-lines](https://github.com/yixiaojiu/stylelint-file-max-lines) - Limit a maximum number of lines per file.
-- [stylelint-force-app-name-prefix](https://github.com/SunHuawei/stylelint-force-app-name-prefix/) - Enforce app name prefix for rules.
-- [stylelint-gamut](https://github.com/fpetrakov/stylelint-gamut) - Enforce standard different color spaces (Pack).
-- [stylelint-group-selectors](https://github.com/ssivanatarajan/stylelint-group-selectors) - Group selectors that have the same set of properties and values.
+- [@isnotdefined/stylelint-plugin](https://www.npmjs.com/package/@isnotdefined/stylelint-plugin) - Disallow obsolete CSS and enforce unit steps (Pack).
+- [@stylistic/stylelint-plugin](https://www.npmjs.com/package/@stylistic/stylelint-plugin) - A collection of stylistic rules that have been removed from Stylelint@16.
```
Comment (jeddy3) [inline README.md:77]:
```suggestion
- [@stylistic/stylelint-plugin](https://www.npmjs.com/package/@stylistic/stylelint-plugin) - Enforce stylistic conventions by porting the rules removed in Stylelint 16.0.0 (Pack)
```

Let's continue the conventions of using:

- "(Pack)" for collections
- descriptions that, where possible, align with our conventions for rule descriptions

File: README.md [hunk 2]
```diff
@@ -67,69 +67,69 @@
...
+- [stylelint-8-point-grid](https://www.npmjs.com/package/stylelint-8-point-grid) - Enforce an 8-point grid.
+- [stylelint-a11y](https://www.npmjs.com/package/stylelint-a11y) - Enforce accessibility standards (Pack).
+- [stylelint-at-rule-no-children](https://www.npmjs.com/package/stylelint-at-rule-no-children) - Disallow block rule declarations inside at-rules.
+- [stylelint-color-format](https://www.npmjs.com/package/stylelint-color-format) - Convert HEX colors to either RGB or HSL.
```
Comment (jeddy3) [inline README.md:81]:
```suggestion
- [stylelint-color-format](https://www.npmjs.com/package/stylelint-color-format) - Enforce HEX, RGB or HSL colors.
```

File: README.md [hunk 3]
```diff
@@ -67,69 +67,69 @@
...
+- [stylelint-declaration-block-no-ignored-properties](https://www.npmjs.com/package/stylelint-declaration-block-no-ignored-properties) - Disallow property values that are ignored.
+- [stylelint-declaration-strict-value](https://www.npmjs.com/package/stylelint-declaration-strict-value) - Enforce variables, functions or keywords for the value of specific properties.
+- [stylelint-file-max-lines](https://www.npmjs.com/package/stylelint-file-max-lines) - Limit a maximum number of lines per file.
+- [stylelint-force-app-name-prefix](https://www.npmjs.com/package/stylelint-force-app-name-prefix) - Enforce app name prefix for rules.
+- [stylelint-gamut](https://www.npmjs.com/package/stylelint-gamut) - Enforce standard different color spaces (Pack).
+- [stylelint-group-selectors](https://www.npmjs.com/package/stylelint-group-selectors) - Group selectors that have the same set of properties and values.
 - [stylelint-high-performance-animation](https://github.com/kristerkari/stylelint-high-performance-animation) - Avoid low-performance animation and transition properties.
```
Comment (jeddy3) [inline README.md:91]:
```suggestion
- [stylelint-high-performance-animation](https://github.com/kristerkari/stylelint-high-performance-animation) - Disallow low-performance animation and transition properties.
```

File: README.md [hunk 4]
```diff
@@ -67,69 +67,69 @@
...
+- [stylelint-no-browser-hacks](https://www.npmjs.com/package/stylelint-no-browser-hacks) - Disallow browser hacks that are irrelevant to the browsers you are targeting.
+- [stylelint-no-indistinguishable-colors](https://www.npmjs.com/package/stylelint-no-indistinguishable-colors) - Disallow colors that are suspiciously close to being identical.
+- [stylelint-no-nested-media](https://www.npmjs.com/package/stylelint-no-nested-media) - Disallow nested `@media` rules.
+- [stylelint-no-restricted-syntax](https://www.npmjs.com/package/stylelint-no-restricted-syntax) - Disallow specified syntax.
+- [stylelint-no-unresolved-module](https://www.npmjs.com/package/stylelint-no-unresolved-module) - Disallow unresolved imports (import-like or `url`).
+- [stylelint-no-unsupported-browser-features](https://www.npmjs.com/package/stylelint-no-unsupported-browser-features) - Disallow CSS that is unsupported by the browsers you're targeting.
+- [stylelint-no-unused-selectors](https://www.npmjs.com/package/stylelint-no-unused-selectors) - Disallow unused selectors.
+- [stylelint-order](https://www.npmjs.com/package/stylelint-order) - Order things like properties.
```
Comment (jeddy3) [inline README.md:103]:
```suggestion
- [stylelint-order](https://www.npmjs.com/package/stylelint-order) - Enforce a specific order for things like properties (Pack).
```

File: README.md [hunk 5]
```diff
@@ -67,69 +67,69 @@
...
+- [stylelint-plugin-defensive-css](https://www.npmjs.com/package/stylelint-plugin-defensive-css) - A Stylelint plugin for enforcing defensive CSS best practices.
```
Comment (jeddy3) [inline README.md:104]:
```suggestion
- [stylelint-plugin-defensive-css](https://www.npmjs.com/package/stylelint-plugin-defensive-css) - Enforce defensive CSS best practices (Pack).
```

File: README.md [hunk 6]
```diff
@@ -67,69 +67,69 @@
...
+- [stylelint-plugin-ecss](https://www.npmjs.com/package/stylelint-plugin-ecss) - Enforce Enduring CSS architecture.
+- [stylelint-plugin-logical-css](https://www.npmjs.com/package/stylelint-plugin-logical-css) - A Stylelint plugin to enforce logical CSS properties, values and units.
```
Comment (jeddy3) [inline README.md:106]:
```suggestion
- [stylelint-plugin-logical-css](https://www.npmjs.com/package/stylelint-plugin-logical-css) - Enforce logical CSS properties, values and units (Pack).
```

File: README.md [hunk 7]
```diff
@@ -67,69 +67,69 @@
...
+- [stylelint-react-native](https://www.npmjs.com/package/stylelint-react-native) - Enforce React Native rules (Pack).
+- [stylelint-rem-over-px](https://www.npmjs.com/package/stylelint-rem-over-px) - Enforce `rem` units over `px` units.
+- [stylelint-rscss](https://www.npmjs.com/package/stylelint-rscss) - Enforce RSCSS conventions (Pack).
+- [stylelint-sass-render-errors](https://www.npmjs.com/package/stylelint-sass-render-errors) - Display Sass render errors and deprecations.
+- [stylelint-scales](https://www.npmjs.com/package/stylelint-scales) - Enforce scales for numeric values (Pack).
+- [stylelint-scss](https://www.npmjs.com/package/stylelint-scss) - SCSS-specific custom rules (Pack).
```
Comment (jeddy3) [inline README.md:112]:
```suggestion
- [stylelint-scss](https://www.npmjs.com/package/stylelint-scss) - Enforce SCSS-specific conventions (Pack).
```

File: README.md [hunk 8]
```diff
@@ -67,69 +67,69 @@
...
+- [stylelint-selector-bem-pattern](https://www.npmjs.com/package/stylelint-selector-bem-pattern) - Enforce BEM patterns for selectors.
+- [stylelint-selector-no-empty](https://www.npmjs.com/package/stylelint-selector-no-empty) - Disallow empty selectors.
+- [stylelint-selector-tag-no-without-class](https://www.npmjs.com/package/stylelint-selector-tag-no-without-class) - Disallow certain tags without a class qualifier in selectors.
+- [stylelint-stylus](https://www.npmjs.com/package/stylelint-stylus) - Stylelint plugin for Stylus.
```
Comment (jeddy3) [inline README.md:116]:
```suggestion
- [stylelint-stylus](https://www.npmjs.com/package/stylelint-stylus) - Enforce Stylus-specific conventions.
```

**Review by jeddy3 — CHANGES_REQUESTED** [review]
> I clicked through all the links, and found some deprecated packages among them. I changed them to new ones.

Thank you for doing that! It must've taken some time.

I've suggested a few changes to the plugin descriptions to make them consistent. Otherwise, it LGTM.

**Review by jeddy3 — APPROVED** [review]
LGTM, thank you!

**Review by ybiquitous — APPROVED** [review]
Great! Thanks a lot! 👏🏼 

---

By the way, this awesome list will be published on our website. Then people will find the list more easily. See https://github.com/stylelint/stylelint.io/issues/370

**Pull request merged**